### DISCUSSION TITLE
Update MKS TFT35 v1.0

### DISCUSSION THREAD

**Discussion by jcbr92** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2627] [General]
Hello I need help with this. I can't install firmware on a MKS TFT35 v1.0
I have followed the guide, but the installation never completes. What can go wrong? Thank you

![TFT](https://user-images.githubusercontent.com/44876703/200117282-b8435f5d-98c8-438a-b05a-a31ca162cd20.png)

**Comment by stef-ladefense** [comment]
me too !

**Comment by stuclem47** [comment]
And me!

**Comment by stef-ladefense** [comment]
corrected 
https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/issues/2719#issuecomment-1438411461
